### ISSUE TITLE
Ignore mockery-generated files: `// Code generated by mockery v1.0.0`

### ISSUE THREAD

**Issue by jirfag** [golangci/golangci-lint#72]

**Comment by alecthomas** [comment]
There is a (sort of?) proposal for standardising on an auto-generated comment format here:
https://github.com/golang/go/issues/13560

It's a bit hard to tell what the outcome is, but it might be worth following that.

**Comment by jirfag** [comment]
thank you, but some tools don't follow this [convention](https://golang.org/s/generatedcode).
Generated code detection was already softened in https://github.com/golangci/golangci-lint/pull/48, now maybe just need to find the line `// Code generated.*`

**Comment by WillAbides** [comment]
I ran into this and found that updating mockery appends "DO NOT EDIT" to the "Code generated..." comment.  This causes it to be in line with https://github.com/golang/go/issues/13560 and thus be treated as generated code by golangci-lint.  It looks like that was added in https://github.com/vektra/mockery/pull/188.

It might still be worth adding this exception because there are still a lot of files out there that were generated by older mockery binaries.
